### DISCUSSION TITLE
Not Able To Download Full Videos After Activating License With Version 9.1.0.44 in Chrome

### DISCUSSION THREAD

**Discussion by mirofunding** [aclap-dev/video-downloadhelper#1297] [Help and Q&A]
### Your question

I was previously able to download videos that are 1:05:27, now when I try to download the same video using the default settings I am only getting 5 min video.  Your help is greatly appreciated.

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"8206db8fe5104c25"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\mcado\\MiroFunding\\Real Estate\\Canadian Mortgage Brokering\\__Pineapple Mortgages\\Lenders\\Reverse Mortgages\\Equitable Bank_Reverse Mortgages\\Training Videos"
        }
    },
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 4,
    "successfull_dl": 2
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Let's have at least one URL of a web page where you have found a video that is exhibiting this problem.

**Comment by mirofunding** [comment]
Herfe is the website site:

https://www.equitablebank.ca/resources/broker-resources/reverse-mortgages-resources/information-training-for-brokers

If you scroll down the page you will see a few videos that are 1hr or more
in length

   1. Reverse Mortgage Training - Marketing to Potential Customers -
   1:04:42 hrs => just tried downloading again a few minutes ago and only
   getting 30 sec


   1. Reverse Mortgage Training – Educating Potential Customers - 1:05:27
   hrs
   2. The Reverse Mortgage Opportunity - 1:12:01 hrs



Thanks,

Mike Cadotte
***@***.***

**Reply by Mattmjs** [reply]
I saw the 1:04:42 video download for about 10% before it stopped, it might start out alright but the speed dropped down
to nothing. Could be throttling the download to a point where it doesn't think there is more to download and stops.
I tried ffmpeg on it's own but it doesn't allow it, either 404 or 403 error.

**Reply by GrampaWildWilly** [reply]
I got the same results.  This behavior of starting out fine & then just quitting reminds me of Pornhub.  It acts the same way.  There, the solution is to just keep retrying.  I've seen Pornhub take 8 or 12 retries before you get what you're trying to download.  But there, we're typically talking about clips of 15 minutes or less.  I don't know if it's worth it to keep retrying this one.  I'm not going to put myself through that pain.  Maybe this is important enough to you to try that.  When I say retry, first I reload VDH from the `Reload addon` button in the settings.  Then I reload the web page.  Then I retry the VDH download.

I decided to look at the details of the last video on the page because it's the longest one:

<html><body>

    
Tab | 95 (incognito: false)
-- | --
Information & Training for Brokers \| Equitable Bank
https://www.equitablebank.ca/resources/broker-resources/reverse-mortgages-resources/information-training-for-brokers  with 12 headers
/content/images/no-thumbnail.png
false
https://www.lightcast.com/embed/playlist_m3u8_vod2.php?code=eyJ2aWRlb19pZCI6IjM5NTI1MSIsInNvdXJjZSI6Im90aGVyc2l0ZXMiLCJjaGFubmVsX2lkIjpmYWxzZSwiZGV2aWNlSWQiOiIxNzM1MjEyOTAxLTg0Y2U0Yzc2ZWFjOTEyMGJmYTU3YTI5ZWU0Y2MwODY0IiwibGNwIjoyfQ==&play_id=1727428206214364
https://hls-vod-media-fs.cdn01.net/aFpXT2xWV1l6Z1RaME1UWTBBek0wWXpOM1ltWmlSRE5sbERNelEyTmtGRE8wa1RPMVVqTnpnSE1fNjA2NDE1NzI3MV9XS1lFS1JUSVdQVlo/encoded/0000395/0395251/video_1880k/913BC90HO.m3u8?source=othersites&device_id=1735212901-84ce4c76eac9120bfa57a29ee4cc0864&lcp=2&client_id=523293257817193003&play_id=1727428206214364
none
Mp4 b:Hls p:hls 4321.267s
unknown at 1888000, fps: ?, q:?, no dim
https://www.lightcast.com/embed/playlist_m3u8_vod2.php?code=eyJ2aWRlb19pZCI6IjM5NTI1MSIsInNvdXJjZSI6Im90aGVyc2l0ZXMiLCJjaGFubmVsX2lkIjpmYWxzZSwiZGV2aWNlSWQiOiIxNzM1MjEyOTAxLTg0Y2U0Yzc2ZWFjOTEyMGJmYTU3YTI5ZWU0Y2MwODY0IiwibGNwIjoyfQ==&play_id=1727428206214364
https://hls-vod-media-fs.cdn01.net/aFpXT2xWV1l6Z1RaME1UWTBBek0wWXpOM1ltWmlSRE5sbERNelEyTmtGRE8wa1RPMVVqTnpnSE1fNjA2NDE1NzI3MV9XS1lFS1JUSVdQVlo/encoded/0000395/0395251/video_640k/913BC90HO.m3u8?source=othersites&device_id=1735212901-84ce4c76eac9120bfa57a29ee4cc0864&lcp=2&client_id=523293257817193003&play_id=1727428206214364
none
Mp4 b:Hls p:hls 4321.267s
unknown at 676000, fps: ?, q:?, no dim
https://www.lightcast.com/embed/playlist_m3u8_vod2.php?code=eyJ2aWRlb19pZCI6IjM5NTI1MSIsInNvdXJjZSI6Im90aGVyc2l0ZXMiLCJjaGFubmVsX2lkIjpmYWxzZSwiZGV2aWNlSWQiOiIxNzM1MjEyOTAxLTg0Y2U0Yzc2ZWFjOTEyMGJmYTU3YTI5ZWU0Y2MwODY0IiwibGNwIjoyfQ==&play_id=1727428206214364
https://hls-vod-media-fs.cdn01.net/aFpXT2xWV1l6Z1RaME1UWTBBek0wWXpOM1ltWmlSRE5sbERNelEyTmtGRE8wa1RPMVVqTnpnSE1fNjA2NDE1NzI3MV9XS1lFS1JUSVdQVlo/encoded/0000395/0395251/video_368k/913BC90HO.m3u8?source=othersites&device_id=1735212901-84ce4c76eac9120bfa57a29ee4cc0864&lcp=2&client_id=523293257817193003&play_id=1727428206214364
none
Mp4 b:Hls p:hls 4321.267s
unknown at 400000, fps: ?, q:?, no dim
https://www.lightcast.com/embed/playlist_m3u8_vod2.php?code=eyJ2aWRlb19pZCI6IjM5NTI1MSIsInNvdXJjZSI6Im90aGVyc2l0ZXMiLCJjaGFubmVsX2lkIjpmYWxzZSwiZGV2aWNlSWQiOiIxNzM1MjEyOTAxLTg0Y2U0Yzc2ZWFjOTEyMGJmYTU3YTI5ZWU0Y2MwODY0IiwibGNwIjoyfQ==&play_id=1727428206214364
https://hls-vod-media-fs.cdn01.net/aFpXT2xWV1l6Z1RaME1UWTBBek0wWXpOM1ltWmlSRE5sbERNelEyTmtGRE8wa1RPMVVqTnpnSE1fNjA2NDE1NzI3MV9XS1lFS1JUSVdQVlo/encoded/0000395/0395251/video_192k/913BC90HO.m3u8?source=othersites&device_id=1735212901-84ce4c76eac9120bfa57a29ee4cc0864&lcp=2&client_id=523293257817193003&play_id=1727428206214364
none
Mp4 b:Hls p:hls 4321.267s
unknown at 192000, fps: ?, q:?, no dim


    
  


</body>
</html>

That's actually kind of a mess.  Here's what it really looks like:

![#01](https://github.com/user-attachments/assets/8b2d9dbc-ea40-4195-8d80-5c90b31a0bb5)

That looks like chicken tracks.  Click the image.  GitHub will open it in its own browser window.  From there, use Ctrl-+ to magnify it.

There's no audio tracks with this.  It's also missing the resolution information.  This site must be doing something really strange.

I am declaring a jubilant defeat & moving on.

**Comment by mirofunding** [comment]
Well thanks for trying.  Maybe it's just a coincidence that before I paid
for my upgrade the video download from the same site worked, now it doesn't


Thanks,

Mike Cadotte
(613) 697-9678
***@***.***
